dotcom-rendering/cdk/lib/renderingStack.ts

```diff
@@ -175,7 +175,7 @@ export class RenderingCDKStack extends CDKStack {
 					metric: latencyMetric,
 					scalingSteps: props.scaling.policy.scalingStepsOut,
 					adjustmentType: AdjustmentType.PERCENT_CHANGE_IN_CAPACITY,
-					evaluationPeriods: 5,
+					evaluationPeriods: 10,
```

Comment:
10 periods is **5 min** IIUC

Reply:
10 periods is 5 min yes (one period is 30 secs) - I am hoping that yeah caching is enough to help here, and that 5 minutes isn't such a big deal, just because our response time is so spiky..